### PR TITLE
Create structure based off of wireframes

### PR DESCRIPTION
**Pull request by danielschifano**
landing-page

### CODE REVIEW

**Comment by kangarang** [comment]
@danielschifano lookin good so far. could you remove the `.DS_Store` files? you can use this as a guide: https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder. lemme know if you get stuck

**Comment by danielschifano** [comment]
> @danielschifano lookin good so far. could you remove the `.DS_Store` files? you can use this as a guide: https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder. lemme know if you get stuck

Done 😄 :shipit:

File: website/index.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Panvala</title>
+  	<meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="css/util.css"/>
+    <link rel="stylesheet" href="css/colors.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
```
Comment (kangarang) [inline website/index.html]: should this be loaded above the custom css?

File: website/index.html [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        </div>
+      </nav>
+    </section>
+
+    <!-- Hero -->
+    <section class="bg-blue">
+      <div class="w-70-l w-80-m w-90 center dt pv5">
+        <div class="dtc v-mid w-60-l w-80-m w-90">
+          <h1 class="white f1 b ma0 mb4">
+            Ethereum can only be sustained by us.</h2>
```
Comment (kangarang) [inline website/index.html]: should this be a `</h1>` closing tag?

**Pull request merged**